package.json

```diff
@@ -3,10 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": ">=18.0.0",
-    "pnpm": "9.0.6"
+    "node": ">=22.0.0",
+    "pnpm": "^9.0.6"
```

Comment:
Do we also need to bump the version defined below under the `packageManager` key?